west-sbom-report.md

```diff
@@ -0,0 +1,104 @@
+> ### WARNING!
+> Some of the files does not contains license information or the license contained in them cannot be detected.<br>
+See [license details](internal_link) for list of files.
+Evaluate them manually to get license information.
+
+
+# Summary
+
+This report was generated from the following inputs:
+* `zephyr/zephyr.elf from build directory /home/doki/work/ncs2/nrf/samples/crypto/ecdsa/build_sbom_test`
+* `files: **/*.c, **/*.h (relative to /home/doki/work/ncs2/nrf/samples/crypto/ecdsa/)`
+* `list of files read from: /home/doki/work/ncs2/sample-list.txt`
```

Comment:
```suggestion
* The `zephyr/zephyr.elf` file from the build directory `/home/doki/work/ncs2/nrf/samples/crypto/ecdsa/build_sbom_test`.
* Files: `**/*.c` and , `**/*.h` (relative to `/home/doki/work/ncs2/nrf/samples/crypto/ecdsa/`).
* A list of files read from: `/home/doki/work/ncs2/sample-list.txt`
```

Reply:
Each list item comes from the script, so I cannot format it by adding \` \`, so everything must be using the same format. I will change the text.